src/content/docs/en/pages/changelog/previous-years/previous-years-changelog.mdx

```diff
@@ -385,7 +385,7 @@ See the updates from previous years below.
 
 * Improvements in the Edge Services and Edge Nodes UIs;
 * Now users can modify Edge Services that are running on Edge Nodes;
-* Users can now create their own [Credentials](/en/documentation/products/deploy/credentials/).
+* Users can now create their own Credentials.
```

Comment:
There is a 'credential' mention here. Validate if it's the right term (in EN/PT)

Reply:
@MarianaAguilera I didn't remove it because it's a changelog, just removed the link. Do you think it'd be better to remove it anyway?